.github/workflows/release.yml

```
@@ -31,6 +31,7 @@ jobs:
       run: poetry build
 
     - name: Publish to PyPI
+      if: ${{ secrets.PYPI_API_TOKEN != '' }}
```

actionlint (1.7.8)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)